DependencyInjection/Compiler/RegisterPluginsPass.php

```diff
@@ -38,4 +38,23 @@ public function process(ContainerBuilder $container)
             }
         }
     }
-}
+
+    /**
+     * @param ContainerBuilder $container
+     * @param string $name The tag name
+     * @return array
+     */
+    protected function findSortedByPriorityTaggedServiceIds(ContainerBuilder $container, $name)
+    {
+        $taggedServices = $container->findTaggedServiceIds($name);
+        uasort(
+            $taggedServices,
+            function ($a, $b) {
+                $a = isset($a[0]['priority']) ? $a[0]['priority'] : 0;
```

Comment:
I've seen other bundles doing exactly this. What happens if someone tags really the same service with 2 tags (same name) and different priorities? Is the plugin registered twice anyways?